libbeat/processors/actions/docs/truncate_fields.asciidoc

```diff
@@ -24,9 +24,9 @@ For example, this configuration truncates the field named `message` to 5 charact
 ------------------------------------------------------------------------------
 processors:
   - truncate_fields:
-    fields:
-    - message
-    max_characters: 5
-    fail_on_error: false
-    ignore_missing: true
+      fields:
+      - message
```

Comment:
Thanks @kiju98! Does this line need to be indented as well?

```suggestion
        - message
```

Reply:
I test and found both
```
processors:
  - truncate_fields:
      fields:
      - message
```
and
```
processors:
  - truncate_fields:
      fields:
        - message
```
work.

I think the latter is more consistent with other processor descriptions.

Reply:
Thanks for testing that! Agreed. I'll push the indentation change and then get this merged and cherry-picked to the applicable branches. Appreciate you opening this PR.